llvm/lib/Target/AMDGPU/AMDGPUAtomicOptimizer.cpp

```diff
@@ -178,6 +178,20 @@ bool AMDGPUAtomicOptimizerImpl::run(Function &F) {
   return Changed;
 }
 
+static bool shouldOptimizeForType(Type *Ty) {
```

Comment:
Name this something that states why this should be handled, not a vague "shouldOptimize"

Reply:
Any better suggestions ?